client/src/views/Game.vue

```diff
@@ -151,8 +151,18 @@
             ></Card>
           </div>
         </v-col>
+
+        <!-- Chat -->
+        <v-col v-show="chatOpen" class="float-chat">
+          <Chat :gameState="gameState"/>
+        </v-col>
       </v-row>
     </v-container>
+
+    <!-- If we allow the Watchers to interact in the chat I'll need to update that on the class='from-me' or 'from-them' -->
+    <div v-if="gameState.status === 'Playing' && gameState.current_player != undefined &&  gameState.player_id === gameState.current_player.id" @click="chatOpen = !chatOpen" class="float-button">
```

Comment:
So, I'm noticing that due to this if check. only the current player can see the chat window. Was that intentional?

that's due to the `gameState.player_id === gameState.current_player.id` 

I see your note here. Is that what you refer to as watchers?

Reply:
I think your changes should work even if they aren't the current player.... but I did put in an issue earlier to Get all of the Player data from the token endpoint.

https://github.com/jak103/uno/issues/167

This would include getting the username of the client based on the JWT. which I believe would keep your `from-me` and `from-them` working even if not the current player.

Reply:
I see what the problem is and I'll change that. I'm trying to not allow the people that are watching to be able to join the chat.
The from-me and from-them are the CSS classes that I assign inside the chat component. The from-them puts the message no the left side with a color while, from-me puts it on the right and blue.

Reply:
@logan-hendricks  I finished this problem and made the chat work for watchers as well.

Reply:
@mcmullinboy15 I will be glad to test this as soon I can. Once I test it, I'll approve it. and I would love for you to merge it in so the merge commit stays with the author.

Reply:
Sweet! Thanks!